libvast/vast/config.hpp.in

```diff
@@ -8,6 +8,8 @@
 #cmakedefine01 VAST_USE_OPENCL
 #cmakedefine01 VAST_USE_OPENSSL
 #define VAST_VERSION "@VAST_VERSION_TAG@"
+#define VAST_SYSCONFDIR "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_SYSCONFDIR@"
```

Comment:
Do we want to make `CMAKE_INSTALL_SYSCONFDIR` configurable via our `configure` script?

Reply:
No.

Reply:
To elaborate: CMake picks this variable up from the environment. Package managers set it according to their needs.